electron_app/src/electron-main.js

```diff
@@ -295,7 +295,9 @@ app.on('ready', () => {
         console.log('No update_base_url is defined: auto update is disabled');
     }
 
-    const iconPath = `${__dirname}/../img/riot.${process.platform === 'win32' ? 'ico' : 'png'}`;
+    // It's important to call `path.join` so we don't end up with the packaged
+    // ASAR in the final path.
+    const iconPath = path.join(__dirname, `../../img/riot.${process.platform === 'win32' ? 'ico' : 'png'}`);
```

Comment:
while we're using path.join might as well make it `path.join(__dirname, '..', '..', 'img', ...)`

Reply:
Okay, fixed!